server/src/engineModel/EngineModel.ts

```diff
@@ -256,6 +256,6 @@ export class EngineModel {
     private enrichYmlClass(yclass: YmlClass): void {
         yclass.data = `id_${yclass.label}`;
         yclass.detail =
-            yclass.extends.length > 0 ? `Class: ${yclass.label} extends ${yclass.extends}` : `Class: ${yclass.label}`;
+            yclass.extends.length > 0 ? `(class) ${yclass.label} extends ${yclass.extends}` : `(class) ${yclass.label}`;
```

Comment:
Huge-ass ternary thingy with `(class) ${yclass.label}` on both sides. You could yank that part out of there to make it clearer what exactly changes according to `yclass.extends.length`. Kinda old, though, but you placed that stuff right under my nose. :p

```ts
yclass.detail = `(class) ${yclass.label}`
if(yclass.extends.length > 0) {
    yclass.detail += ` extends ${yclass.extends}`
}
```

LOL it’s only while writing this that I understood what this was doing exactly (I wasn’t paying much attention).

Reply:
Your version is better.